cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/FileStreamAdmin.java

```diff
@@ -193,7 +187,9 @@ public void dropAllInNamespace(final NamespaceId namespace) throws Exception {
     });
 
     for (final Location location : locations) {
-      doDrop(namespace.stream(StreamUtils.getStreamNameFromLocation(location)), location);
+      StreamId streamId = namespace.stream(StreamUtils.getStreamNameFromLocation(location));
+      ensureAccess(streamId, Action.ADMIN);
```

Comment:
This is related to https://github.com/caskdata/cdap/pull/9337#discussion_r133108480